<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XplayerClass</class>
 <widget class="QMainWindow" name="XplayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Xplayer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Xplayer.qrc">
    <normaloff>:/Xplayer/Resources/logo.ico</normaloff>:/Xplayer/Resources/logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="VideoWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileBtn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>40</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:!hover{border-image: url(:/Xplayer/Resources/open_normal.png);}
QPushButton:hover{border-image: url(:/Xplayer/Resources/open_hot.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playAvBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>500</y>
      <width>40</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:!hover{border-image: url(:/Xplayer/Resources/play_normal.png);}
QPushButton:hover{border-image: url(:/Xplayer/Resources/play_hot.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="playtime">
    <property name="geometry">
     <rect>
      <x>691</x>
      <y>491</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>000:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="sp">
    <property name="geometry">
     <rect>
      <x>733</x>
      <y>491</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="totaltime">
    <property name="geometry">
     <rect>
      <x>745</x>
      <y>491</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>000:00</string>
    </property>
   </widget>
   <widget class="XSlider" name="playSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>781</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videowidget.h</header>
  </customwidget>
  <customwidget>
   <class>XSlider</class>
   <extends>QSlider</extends>
   <header>xslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Xplayer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>openFileBtn</sender>
   <signal>clicked()</signal>
   <receiver>XplayerClass</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>XplayerClass</receiver>
   <slot>sliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>815</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>XplayerClass</receiver>
   <slot>sliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>818</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playAvBtn</sender>
   <signal>clicked()</signal>
   <receiver>XplayerClass</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>-30</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open()</slot>
  <slot>sliderPress()</slot>
  <slot>sliderRelease()</slot>
  <slot>play()</slot>
 </slots>
</ui>
